### PR TITLE
Task_3

### PR DESCRIPTION
**Pull request by AndreiKononov**

### CODE REVIEW

File: src/Components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons';
+import './Dropdown.scss';
+
+interface DropdownConfig {
```
Comment (kuzanatoliy) [inline src/Components/Dropdown/Dropdown.tsx]:
```suggestion
interface IDropdownProps {
```
Just could be little bit more clear

File: src/Components/Hero/Hero.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
+@import 'src/scss/variables';
+
+.hero {
+  position: relative;
+  height: 396px;
+  padding: 0 #{$layout-side-padding};
+  background-image: url('../../img/header.png');
```
Comment (kuzanatoliy) [inline src/Components/Hero/Hero.scss]: I would use absolute path

**Review by kuzanatoliy — COMMENTED** [review]
Please add one of the general linter config as google or airbnb
Also I would create public directory for static content

**Comment by AndreiKononov** [comment]
> Please add one of the general linter config as google or airbnb Also I would create public directory for static content

Done.

File: src/Components/Dropdown/Dropdown.tsx (hunk 1 above)
Comment (AndreiKononov) [reply src/Components/Dropdown/Dropdown.tsx]: Done.

File: src/Components/Hero/Hero.scss (hunk 2 above)
Comment (AndreiKononov) [reply src/Components/Hero/Hero.scss]: Not sure, how I can use it here, to be honest.
Relative one is possible as well.
https://developer.mozilla.org/en-US/docs/Web/CSS/url

**Pull request merged**